src/utils/withTheme.jsx

```diff
@@ -0,0 +1,16 @@
+import { Component } from 'react';
```

Comment:
I'm pretty sure we don't need this and that the styleguide should render its own `MuiThemeProvider`.

Reply:
Ah, it turns out I can just do it by overwriting the Wrapper field of the styleguide.